index.html

```diff
@@ -59,122 +87,60 @@ <h3>let's connect</h3>
-              <span class="subheading-list">2015</span>
-            </div>
-            <div class="card-content">
-              <p>
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required.
-              </p>
-            </div>
-            <ul class="web-languages">
-              <li><a href="/">html</a></li>
-              <li><a href="/">css</a></li>
-              <li><a href="/">javascript</a></li>
-            </ul>
-            <button class="see-project">See project</button>
-          </div>
+        <div class="subheading">
+          <span class="canopy">canopy</span>
+          <img src="./Icons/Counter.png" alt="" />
+          <span class="subheading-list">back end dev</span>
+          <img src="./Icons/Counter.png" alt="" />
+          <span class="subheading-list">2015</span>
         </div>
-      </section>
-
-      <section class="card-works">
-        <div class="card">
-          <img src="./Icons/Snapshoot Portfolio (3).png" alt="" />
-          <div class="card-description">
-            <h2 class="card-title">Multi-Post Stories</h2>
-            <div class="subheading">
-              <span class="canopy">canopy</span>
-              <img src="./Icons/Counter.png" alt="" />
-              <span class="subheading-list">back end dev</span>
-              <img src="./Icons/Counter.png" alt="" />
-              <span class="subheading-list">2015</span>
-            </div>
-            <div class="card-content">
-              <p>
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required.
-              </p>
-            </div>
+        <img class="detail-img" src="./Icons/Snapshoot Portfolio.png" alt="">
+        <div class="details-content">
+          <p class="description">
+            Lorem ipsum dolor sit, amet consectetur
+             adipisicing elit. Veritatis distinctio
+              vero cumque, aspernatur veniam molestiae
+               quibusdam repellat. Labore expedita
+                voluptate earum. Nam assumenda
+                 dolorum qui esse soluta quod, minima nemo!
+          </p>
+          <div class="details-right">
             <ul class="web-languages">
               <li><a href="/">html</a></li>
               <li><a href="/">css</a></li>
               <li><a href="/">javascript</a></li>
             </ul>
-            <button class="see-project">See project</button>
+            <div class="details-links">
+              <a href="/">
+                <button>
+                  <p>See live</p> <img src="./Icons/social/Icon - Export.svg" alt="">
+                </button>
+              </a>
+              <a href="/">
+                <button>
+                  <p>See source</p> <img src="./Icons/social/Frame.svg" alt="">
+                </button>
+              </a>
+            </div>
           </div>
         </div>
-      </section>
+      </div> -->
```

Comment:
- [ ] It's a good thing you currently load this part of your portfolio dynamically. How about removing the whole of this completely? Commenting old pieces of code in a workspace makes the code looks kinda messy :hammer_and_wrench: